_posts/2019-09-10-jupyter-on-dask.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: Co-locating a Jupyter Server and Dask Scheduler
+author: Matthew Rocklin
+tags: [HPC]
+theme: twitter
+---
+{% include JB/setup %}
+
+If you want, you can have Dask set up a Jupyter notebook server for you,
+co-located with the Dask scheduler.  There are many ways to do this, but this
+blog post lists two.
+
+### First, why would you do this?
+
+Sometimes people inside of large institutions have complex deployment pains.
```

Comment:
complex deployment ~~pain~~ plans

Also, institutions may have a large number of distinct, isolated cloud accounts where data cannot be moved.  We need to be able to quickly bring in ephemeral compute infrastructure from the ground up.